content/docs/setup/kubernetes/additional-setup/mesh-expansion/index.md

```diff
@@ -175,49 +196,115 @@ The following example shows accessing a service running in the Kubernetes cluste
     the cluster service from the VM, as in the example below:
 
     {{< text bash >}}
-$ sudo echo "10.55.246.247 productpage.bookinfo.svc.cluster.local" >> /etc/hosts
-$ curl productpage.bookinfo.svc.cluster.local:9080
+$ echo "10.55.246.247 productpage.default.svc.cluster.local" | sudo tee -a /etc/hosts
+$ curl -v productpage.default.svc.cluster.local:9080
+< HTTP/1.1 200 OK
+< content-type: text/html; charset=utf-8
+< content-length: 1836
+< server: envoy
 ... html content ...
     {{< /text >}}
 
+The `server: envoy` header indicates the traffic indeed is intercepted by sidecar.
+
 ## Running services on a mesh expansion machine
 
-You add VM services to the mesh by configuring a
-[`ServiceEntry`](/docs/reference/config/networking/v1alpha3/service-entry/). A service entry lets you manually add
+1. We first setup a HTTP server on the VM instance, serving HTTP traffic on port 8080.
+
+    {{< text bash >}}
+    $ gcloud compute ssh ${GCE_NAME}
+    $ python -m SimpleHTTPServer 8080
+    {{< /text >}}
+
+1. Determine the VM instance IP address. For example, commands to find GCE instance IP are as below.
+
+    {{< text bash >}}
+    $ export GCE_IP=$(gcloud --format="value(networkInterfaces[0].networkIP)" compute instances describe  ${GCE_NAME})
+    $ echo ${GCE_IP}
+    {{< /text >}}
+
+1. Configure Service Discovery for VM by `ServiceEntry`. You add VM services to the mesh by configuring a
+[`ServiceEntry`](/docs/reference/config/networking/v1alpha3/service-entry/). A `ServiceEntry` lets you manually add
 additional services to Istio's model of the mesh so that other services can find and direct traffic to them. Each
 `ServiceEntry` configuration contains the IP addresses, ports, and labels (where appropriate) of all VMs exposing a
 particular service, as in the following example.
 
-{{< text bash yaml >}}
-$ kubectl -n test apply -f - <<EOF
-apiVersion: networking.istio.io/v1alpha3
-kind: ServiceEntry
-metadata:
-  name: vm1
-spec:
-   hosts:
-   - vm1.test.svc.cluster.local
-   ports:
-   - number: 80
-     name: http
-     protocol: HTTP
-   resolution: STATIC
-   endpoints:
-    - address: 10.128.0.17
-      ports:
-        http: 8080
-      labels:
-        app: vm1
-        version: 1
-EOF
+
+    {{< text bash yaml >}}
+    $ kubectl -n ${SERVICE_NAMESPACE} apply -f - <<EOF
+    apiVersion: networking.istio.io/v1alpha3
+    kind: ServiceEntry
+    metadata:
+    name: vmhttp
+    spec:
+    hosts:
+    - vmhttp.${SERVICE_NAMESPACE}.svc.cluster.local
+    ports:
+    - number: 8080
+        name: http
+        protocol: HTTP
+    resolution: STATIC
+    endpoints:
+        - address: ${GCE_IP}
+        ports:
+            http: 8080
+        labels:
+            app: vmhttp
+            version: "v1"
+    EOF
+    {{< /text >}}
+
+1. Workloads in a Kubernetes cluster need a DNS mapping to resolve the service running on the VM.
```

Comment:
```suggestion
1. The workloads in a Kubernetes cluster need a DNS mapping to resolve the the domain names of VM services. To 
    integrate the mapping with you own DNS system, use `istioctl register` and creates a Kubernetes `selector-less` 
    service, for example:
```

Reply:
Although not matter in markdown, why newline in these cases? I searched a bit on [style guide](https://preliminary.istio.io/about/contribute/style-guide) but didn't find any recommended line length.

Reply:
This is a recommendation from GitHub. Up to 80 characters are properly shown in a single line for review.